bundle/Controller/ContentEditController.php

```diff
@@ -117,7 +110,7 @@ public function createWithoutDraftAction($contentTypeIdentifier, $language, $par
      * @param string $toLanguage
      * @param \Symfony\Component\HttpFoundation\Request $request
      *
-     * @return \EzSystems\RepositoryForms\Content\View\ContentCreateDraftView|\Symfony\Component\HttpFoundation\Response
+     * @return \EzSystems\RepositoryForms\Content\View\ContentCreateDraftView|Response
```

Comment:
There's a lot of inconsistency in PhpDoc type hinting. Please decide to use either FQCN or short name.

Reply:
To my knowledge, we use fully-qualified namespaces.

While real IDEs will help navigate with short names, you don't get that option when reviewing code on github. I think it's a very valid argument.

Reply:
Depends on the repository. AdminUI uses short names. Is there a way to enforce it with PHP CS Fixer? @alongosz do you know?

Reply:
> Is there a way to enforce it with PHP CS Fixer?

No, I already looked for it some time ago. Checked just now - nothing changed - we probably would need to contribute... :)

Reply:
Ok. I've changed to FQCN.